scripts/ddd/ddd

```diff
@@ -95,7 +95,11 @@ restore_mapping() {
 
 	# If we have stored USB_MAP, reload it
 	if [ -f ${DDD_MAP} ]; then
-		source ${DDD_MAP}
+        source ${DDD_MAP}
```

Comment:
I could be very mistaken, but as far as I could tell it was the only way to actually load the map - the way it was I think it was just loading the map as a variable local to the function. (And I couldn't find a way to copy bash maps any other way than value by value) If there's a better way though, I'm all for it!